clients/admin-ui/src/features/system/DataFlowsAccordionItem.tsx

```diff
@@ -127,9 +133,7 @@ const DataFlowsAccordionItem = ({ classifyDataFlows, type }: Props) => {
     <AccordionItem p={0} data-testid={`accordion-item-${type}`}>
```

Comment:
I thought about updating `type` to be the desired value, but saw Cypress relying on these `testid`s so thought it better not to change.

Reply:
Can the cypress tests be updated to use the new values?

Reply:
They could be, it's fairly moot and leaving them as-is makes this diff a lot cleaner

Reply:
That seems reasonable 👍